config/services.yaml

```diff
@@ -23,6 +23,9 @@ parameters:
 
     TYPESENSE_KEY: '%env(TYPESENSE_KEY)%'
     TYPESENSE_URL: '%env(TYPESENSE_URL)%'
+    TYPESENSE_EMBED_MODEL: '%env(TYPESENSE_EMBED_MODEL)%'
```

Comment:
You should also put the env(...): "" to have a default value (and in this case you can remove the empty values from .env.)